examples/air_temperature_gullingen_skisenter-parent_meta.json

```diff
@@ -0,0 +1,44 @@
+{
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [
+            59.4128,
+            6.4697
+        ]
+    },
+    "properties": {
+        "title": "Air temperature observations from weather station Gullingen , ROGALAND, Norge, (WIGOS ID: 0-578-0-46220, station ID: 46220).",
+        "data_id": "8a0744b7-f0dc-4929-8700-f1285e76df36",
```

Comment:
Should be on the form "\<naming_authority\>:\<uuid\>" now, right? Also it should not be the same as the `metadata_id`, which we agreed should refer to the parent dataset metadata?